go/bind/keybase.go

```diff
@@ -51,8 +55,37 @@ var kbfsConfig libkbfs.Config
 var initMutex sync.Mutex
 var initComplete bool
 
+type Person struct {
```

Comment:
Let's move all this stuff to a new file in `bind`, call it `notifications.go` or something.